### ISSUE TITLE
Add support for text/event-stream

### ISSUE THREAD

**Issue by murrayju** [Huachao/vscode-restclient#493]
It would be great to have support for [Server Sent Events (SSE)](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events). Such APIs respond with a content type of `text/event-stream`, which is a simple text format for the server to emit data to the client.

The biggest difference between this and other types of requests is that the response stays open indefinitely. I'd like to see the response data in its own tab (separate from other types of requests), which renders the data "live" as it is received. Probably requires some sort of disconnect action (could be just closing the tab).

A very basic use case would be to open such an event stream, then make some other requests that trigger some events. I'd like to see the responses from these requests in one tab, and any event data in another.

**Comment by Huachao** [comment]
@murrayju nice suggestion, I will add this to the backlog

**Comment by vasanthdeveloper** [comment]
Is this featured added? It's been a year since it was opened... 🙄

**Comment by kaneshir** [comment]
yes please give an update on when this will be added as I have the same use case @murrayju is describing.

**Comment by Med-H** [comment]
any news on the event sourcing ?

**Comment by Hermholtz** [comment]
Oh I wanted it today so badly 😄

**Comment by waltcow** [comment]
need this feature

**Comment by deybith** [comment]
Hoping to have soon

**Comment by johnseabird** [comment]
Any news about this?

**Comment by ferrerogg** [comment]
Need this feature :)

**Comment by Puliczek** [comment]
I need that :)

**Comment by thomas-kowalski-s-management-services** [comment]
Would be a great feature.   @Huachao Would you be interested in a PR? If yes, any pointers to get startet?
